genesyscloud/tfexporter/resource_genesyscloud_tf_export_test.go

```diff
@@ -43,52 +43,52 @@ type QueueExport struct {
 	ResourceName string `-`
 }
 
+// TestAccResourceTfExport does a basic test check to make sure the export file is created.
+func TestAccResourceTfExport(t *testing.T) {
+	var (
+		exportTestDir   = "../../.terraform" + uuid.NewString()
+		exportResource1 = "test-export1"
+		configPath      = filepath.Join(exportTestDir, defaultTfJSONFile)
+		statePath       = filepath.Join(exportTestDir, defaultTfStateFile)
+	)
 
-// func TestAccResourceTfExport(t *testing.T) {
```

Comment:
This test case was commented out.  I saw no reason to comment it out and added it back in.